### PR TITLE
Updated Readme file to work properly with Ubuntu 22.04

### PR DESCRIPTION
**Pull request by yvrob**
Installation with Ubuntu 22.04 did not work because some conda packages were not compatible anymore.
Followed instructions on the docker installation, and tested it on a clean Ubuntu 22.04 system.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -108,10 +108,13 @@ To install Cyclus and its dependencies onto a clean Ubuntu machine (tested on 18
 
 - ``conda config --add channels conda-forge``
 
-- ``conda install -y openssh gxx_linux-64 gcc_linux-64 cmake make docker-pycreds git xo
-  python-json-logger python=3.6 glibmm glib=2.56 libxml2 libxmlpp libblas libcblas
-  liblapack pkg-config coincbc=2.9 boost-cpp hdf5 sqlite pcre gettext bzip2 xz
-  setuptools nose pytables pandas jinja2 cython==0.26 websockets pprintpp``
+- ``conda create -n cyclus python=3.8``
```
Comment (gonuke) [inline README.rst:111]: Can you try this with python 3.11?  That is what works for our automated testing.

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks @yvrob for this contribution!  I wonder if you've tried with newer python versions?  I know 3.6 is too old, but we test with 3.11 in conda successfully.

**Comment by yvrob** [comment]
Dear Paul,

Thank, you I will try it and let you know.

Best,
Yves Robert

**Comment by yvrob** [comment]
Dear Paul,

Thank you very much, it should be updated now.

Best regards,
Yves

**Comment by gonuke** [comment]
Thanks for providing this update @yvrob - I'll merge them in shortly.  Hopefully, we can issue a new release soon and include even more extensive updates to aid new users installation.

**Review by gonuke — CHANGES_REQUESTED** [review]
It look like we need one other change, after all.  We require all PRs to include an update to the `CHANGELOG.rst` file, with mention of the PR number.  Please make a new addition to this file and add that file to this PR.

**Comment by yvrob** [comment]
Dear Paul,

Thank you for pointing this out. I made the update. Please let me know if
everything looks good now.

Best regards,

Yves

**Pull request merged**